src/NATS.Client.KeyValueStore/INatsKVContext.cs

```diff
@@ -4,6 +4,11 @@ namespace NATS.Client.KeyValueStore;
 
 public interface INatsKVContext
 {
+    /// <summary>
+    /// Provides access to the JetStream context associated with the Key-Value Store operations.
+    /// </summary>
+    INatsJSContext Context { get; }
```

Comment:
This makes sense to me, but wondering what you think about folks using code like

```cs
var context = client.CreateKeyValueStoreContext();
...
var jsContext = context.Context;
```

I don't mind it, but the thought came to mind and figured I'd mention it.

Reply:
What about `JetStreamContext` or just `JetStream`?
```csharp
var context = client.CreateKeyValueStoreContext();
...
var jsContext = context.JetStreamContext;
...
var jsContext = context.JetStream;
```

Reply:
I like `JetStreamContext`, if there are no conventions in play. The `INatsObjStore` version should probably share the name, if that changes things.

Reply:
done